package.json

```diff
@@ -10,6 +10,7 @@
   "author": "Shashank",
   "license": "ISC",
   "dependencies": {
+    "dotenv": "^16.0.3",
```

Comment:
https://dev.to/numtostr/comment/k026#:~:text=dotenv%20is%20a%20dev%20dependency.

Reply:
sure @shawshankkumar , I'll shift dotenv to devDependencies